node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+const files = process.argv.splice('2');
+async function concatFiles() {
+  try {
+    const promiseArray = await Promise.all(
+      files.map((element) => readFile(element, { encoding: 'utf8' }))
```

Comment:
As a shortcut you can just pass 'utf8' as the second parameter:
`readFile(element, 'utf8');`